svthalia/concrexit
# Add placeholders for discord and online locations

Closes #1465 

### Summary
Add placeholders for discord and online locations.

### How to test
Steps to test the changes you made:
1. Go to an event and set the location for the minimap to 'online'
2. Check that the correct placeholder is shown
3. Repeat the steps above for the location text 'discord'
